install/helm/open-match/values-production.yaml

```diff
@@ -230,6 +230,10 @@ open-match-customize:
   # function:
   #   image: [YOUR_MMF_IMAGE]
 
+# Controls if users need to install open-match-override ConfigMap.
+open-match-override:
+  enabled: false
+
```

Comment:
This comment is actually for L299. The version should be updated there as well

Reply:
Thanks for pointing 👍🏻